root/static/scripts/common/components/ArtistRoles.js

```diff
@@ -24,14 +25,11 @@ type Props = {|
 
 const ArtistRoles = ({relations}: Props) => (
   <ul>
-    {relations.map(r =>(
+    {relations.map(r => (
       <li key={r.entity.id}>
         {l('{artist} ({roles})', {
           artist: <EntityLink entity={r.entity} />,
-          roles: r.roles.length > 1
-            // $FlowFixMe
-            ? commaOnlyList(uniq(r.roles))
-            : r.roles[0],
+          roles: commaOnlyList(uniq(localizeArtistRoles(r.roles))),
```

Comment:
Got rid of the if here because AFAICT commaOnlyList will still give a no comma one element list if there's just one, so I don't see what we're winning by having this here.

Reply:
That's correct, I just figured the majority would only have a single role, so optimized for the common case.

Reply:
I'm not sure I understand whether you think we should keep the conditional or not :)

Reply:
It was premature optimization, so no, it's fine to remove. :)